hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/volume/TestCapacityVolumeChoosingPolicy.java

```diff
@@ -94,9 +102,9 @@ public void setup() throws Exception {
   @AfterEach
   public void cleanUp() {
     volumes.forEach(HddsVolume::shutdown);
-    FileUtil.fullyDelete(new File(VOLUME_1));
-    FileUtil.fullyDelete(new File(VOLUME_2));
-    FileUtil.fullyDelete(new File(VOLUME_3));
+    FileUtil.fullyDelete(new File(volume1));
+    FileUtil.fullyDelete(new File(volume2));
+    FileUtil.fullyDelete(new File(volume3));
```

Comment:
Making `baseDir` non-`static`, and initializing `volumeX` in `@BeforeEach`, we could remove the need for `FileUtil.fullyDelete` in `@AfterEach`.